modules/installation-vsphere-config-yaml.adoc

```diff
@@ -26,61 +26,87 @@ parameters.
 
 [source,yaml]
 ----
+additionalTrustBundlePolicy: Proxyonly
 apiVersion: v1
 baseDomain: example.com <1>
 compute: <2>
-- hyperthreading: Enabled <3>
-  name: worker
+- architecture: amd64
+  hyperthreading: Enabled <3>
+  name: <worker_node>
+  platform: {}
   replicas: 0 <4>
 controlPlane: <2>
+  architecture: amd64
   hyperthreading: Enabled <3>
-  name: master
+  name: <parent_node>
+  platform: {}
   replicas: 3 <5>
 metadata:
+  creationTimestamp: null
   name: test <6>
+networking:
+...
+...
```

Comment:
Are there supposed to be two ellipses here, or should one of these be removed?

Reply:
Swapped out for the triple hyphen.